filebeat/input/kafka/kafka_integration_test.go

```diff
@@ -357,3 +451,27 @@ func writeToKafkaTopic(
 		t.Fatal(err)
 	}
 }
+
+func run(t *testing.T, cfg *common.Config, client *beattest.ChanClient) (func(), *kafkaInput) {
```

Comment:
Please change the order of the returned value. The cancel function is usually the second value.

Reply:
Fixed in https://github.com/elastic/beats/pull/27335/commits/ad521cd749fef1e5cfac8ac75eca3325d3e5cb64